index.html

```diff
@@ -1,373 +1,404 @@
-                                </a>
-                            </li>
-                        
-                        </ul>
-                        <div class="see-project get-resume">
-                            <button type="button" class="p-text">
-                                Get my resume
-                            </button>
-                        </div>
-                    </div>
-                    <ul class="abt-self">
-                        <li class="arrow1">
-                            <p>Languages</p>
-                            <img alt="arr1" src="images/arrow-img.png">
-                        </li>
-                        <li class="code-icons">
-                          <ul class="code-list">
-                            <li>
-                              <img alt="js-icon" src="images/js-img.png">
-                              <p>Javascript</p>
-                            </li>
-                            <li>
-                              <img alt="html-icon" src="images/html-img.png">
-                              <p>HTML</p>
-                            </li>
-                            <li>
-                              <img alt="css-icon" src="images/css-img.png">
-                              <p>CSS</p>
-                            </li>
-                          </ul>
-                        </li>
-                        <li class="arrow1">
-                            <p>Frameworks</p><img alt="arr2" src="images/side-arrow.png">
-                        </li>
-                        <li class="arrow1">
-                            <p>Skills</p><img alt="arr3" src="images/side-arrow.png">
-                        </li>
-                    </ul>
-            </section>
-        </div>
+      </div>
     </main>
-</body>
-</html>
\ No newline at end of file
+    <!-- contact-form -->
+    <section>
+        <div class="form-container" >
+          <div class="background"></div>
+          <div class="head-backgrond">
+            <div class="flex-column">
+              <h2 class="info-head">Contact me</h2>
+              <p class="main-text">
+                if you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I'd
+                love to help with it.
+              </p>
+            </div>
+          </div>
+          <form
+            action="https://formspree.io/f/xknegrdk"
+            method="post"
+            class="form"
+          >
+            <input type="text" name="name" maxlength="30" placeholder="Name" />
+            <input type="email" name="email" maxlength="30" placeholder="Email" />
+            <textarea name="message" maxlength="500" placeholder="Write your message here..."></textarea>
+            <button type="submit" class="btn">
+              <span>Get in touch</span>
+            </button>  
+          </form>
+        </div> 
+      </section>
+  </body>
+</html>
```

Comment:
- [ ] There is an overflow on your project which is causing the contact form to not be properly aligned in mobile view. Kindly fix it.

Your Design
![image](https://user-images.githubusercontent.com/81932478/186667123-3e9eb3c6-d546-413b-8bf9-593799c52a51.png)

Figma Design
![image](https://user-images.githubusercontent.com/81932478/186667303-339228f4-7b0d-4bbc-bc15-23deaad9be46.png)

- [ ] Notice the gray space around your design